### ISSUE TITLE
table of contents toggle breaks layout on at least some browsers

### ISSUE THREAD

**Issue by rfindler** [racket/scribble#318]
[This commit](https://github.com/racket/scribble/commit/a38c141bfa4cbbd72967882025cfdde6cf5936ad) causes the layout of the "top prev up next" section of the documentation to be garbled for me (on Safari on a mac, with a full-width browser). It looks okay in firefox (same window size, same os), tho.

Here's what I see:

<img width="221" alt="Screen Shot 2021-10-06 at 9 44 24 AM" src="https://user-images.githubusercontent.com/399024/136232927-7d8bbe56-4ff5-4184-8a00-31a65d220ad0.png">

and here's what it looks like when I revert the commit, rebuild, and reload the page:

<img width="221" alt="Screen Shot 2021-10-06 at 10 18 56 AM" src="https://user-images.githubusercontent.com/399024/136233154-75d927cd-5bee-49c6-8fcd-976a1d077c93.png">

**Comment by sorawee** [comment]
Ping @jryans in case you know what's going on.

**Comment by jryans** [comment]
Ah, thanks for the ping, I'll take a look!

**Comment by jryans** [comment]
I can indeed reproduce the issue here, and it seems to only affect desktop Safari somehow. Anyway, I'll work on a fix.

**Comment by sorawee** [comment]
Did this PR make to 8.3? docs.racket-lang.org currently looks like this on Firefox.

<img width="226" alt="Screen Shot 2021-11-08 at 3 28 39 AM" src="https://user-images.githubusercontent.com/9099577/140734347-94f1cf99-3bc9-43cf-ac84-0eb1a707e1e9.png">

Note that I don't notice this problem in `raco docs` which is currently at git HEAD.

**Comment by jryans** [comment]
@sorawee Yes, the above markup change has been included correctly in the deployed markup for 8.3.

I believe the visual issue you're seeing is related to a short-lived caching bug with the deployment, as this morning I noticed that the CSS and JS files on `docs.racket-lang.org` didn't appear to be updated, but when I added a random query string (e.g. `?123`), the newer version came through. After a few hours, the updated files eventually came through as part of a regular page load.

On my side, things are now looking correct here.

* Firefox

<img width="225" alt="image" src="https://user-images.githubusercontent.com/279572/140735722-66b7c3d3-349b-4817-98a2-c0a6aa36db8d.png">

* Safari

<img width="225" alt="image" src="https://user-images.githubusercontent.com/279572/140735811-b4832d91-5bb0-4cbd-9d32-cdeea0b00d05.png">

If you're still seeing issues after a few hours, I think it would be good to open a new issue to track this.
